spec.md

```diff
@@ -1495,6 +1613,106 @@ For success responses, the following fields are defined:
 }
 ```
 
+## Adapting Service Binding Credentials
+
+If the endpoints of a Service Instance are from an Application point of view
+are different to the endpoints provided in the binding credentials, then the
+binding credentials MUST be adapted before the Platform provides them to
+the Application.
+
+The Platform requests the corrected credentials from the Service Broker by
+providing the original credentials and a table that maps the original
+endpoints to the new endpoints.
+
+### Request
+
+##### Route
+
+`PUT /v2/service_instances/:instance_id/service_bindings/:binding_id/credentials`
```

Comment:
Wouldn't a `POST` to the path `/v2/service_instances/:instance_id/service_bindings/:binding_id/update_credentials` be preferable?
`PUT` to the path `/v2/service_instances/:instance_id/service_bindings/:binding_id/credentials` might imply an update of the resource credentials (which exists in the binding) and also the possibility to do a `GET` `/v2/service_instances/:instance_id/service_bindings/:binding_id/credentials`.